### ISSUE TITLE
The "Helpful" button's pressed state is not being announced by the screen reader: A11y_Visual Studio Code Client_Usability

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode#286696]
### GitHub Tags:
#A11yTCS; #A11ySev3;#Visual Studio Code Client;#BM_VisualStudioCodeClient_Win32_JULY2024; #DesktopApp; #Win32; #Benchmark;#A11yMAS;#WCAG4.1.2;#Name Role Value;

### Environment and OS details:
Application: VS Code
Version: 1.108.0-insider (user setup)

### Reproduction Steps:

1. Open Visual studio code insider.
2. Tab to Source Controls (or use Ctrl+Shift+G) and open it. 
3. Tab on Copilot Code Review button and select it. 
4. Tab to Helpful control and select it. 
6. Verify whether screen reader is announcing the pressed state or not. 

### Actual Result:
The "Helpful" button's pressed state is not being announced by the screen reader.

### Expected Result:
When the "Helpful" button is toggled:
Pressed State: The screen reader should announce "Helpful button pressed."

### User Impact:
Users relying on screen readers may not understand whether the button is active or inactive, leading to confusion and usability challenges.

### Attachment:

https://github.com/user-attachments/assets/880c7a49-8106-4a3d-95af-c5a39bc17870

**Comment by alexr00** [comment]
None of our toolbars do this. Closing as wontfix.
